frame.js

```diff
@@ -0,0 +1,26 @@
+class Frame {
+  constructor() {
+    this.rolls = [];
+    this.bonus = 0;
+  }
+
+  addRoll(points) {
+    this.rolls.push(points);
+  }
+
+  totalScore() {
+    const reducer = (previousValue, currentValue) => previousValue + currentValue;
```

Comment:
nice - keeping code very readable by separating out the 'reducer' from the reduce function